apps/point-of-sale/src/services/ApiService.ts

```diff
@@ -1,6 +1,6 @@
 import { ApiService } from '@sudosos/sudosos-frontend-common';
 
 const apiService = new ApiService(
-  import.meta.env.VITE_APP_API_BASE
+  window.location.origin + "/api/v1"
```

Comment:
This will not work in the POS, right? As it will result in `sudopos(.test).gewis.nl/api/v1` and not `sudosos(.test).gewis.nl/api/v1`?

Reply:
yes, I had one neuron firing in my brain while helping Job that also realised this, I tactically ignored it. I think the cleanest is to fix this is the nginx routing? Otherwise we are guaranteed to be back where we started. But we will have to maintain two nginx routing then

Reply:
Yes, fixing it via NGINX would be the easiest. However, I do not know how well it will play with CORS?

Reply:
Try in test I suppose